test/index.test.js

```diff
@@ -176,6 +176,28 @@ describe('Mixpanel', function() {
         analytics.called(window.mixpanel.people.set, { trait: true });
       });
 
+      it('should set super properties from the context.mixpanel.superProperties object if setAllTraitsByDefault is false', function() {
+        mixpanel.options.setAllTraitsByDefault = false;
+        analytics.identify(123, { accountStatus: 'Paid', subscribed: true }, { Mixpanel: { superProperties: ['accountStatus', 'subscribed'] } });
```

Comment:
lets include a case that passes more traits than those included in the MP.superProperties array so that we can test that they're omitted correctly